test/tests.js

```diff
@@ -1053,6 +1070,30 @@ describe('pasync', function() {
 		}).catch(done);
 	});
 
+	it('memoize', function(done) {
+		var myFunc = function(up, down, no) {
+			return Promise.resolve(up * down * no);
+		};
+		var myMemo = pasync.memoize(myFunc);
+		myMemo(2, 4, 3).then(function(result) {
+			expect(result).to.equal(24);
+			done();
```

Comment:
Nest this, a couple times with different/same arguments, so we can check if it's actually being memoized.